sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -243,11 +251,13 @@ static bool _gnrc_rpl_check_DIS_validity(gnrc_rpl_dis_t *dis, uint16_t len)
     return false;
 }
 
-void gnrc_rpl_recv_DIS(gnrc_rpl_dis_t *dis, ipv6_addr_t *src, ipv6_addr_t *dst, uint16_t len)
+void gnrc_rpl_recv_DIS(gnrc_rpl_dis_t *dis, kernel_pid_t iface, ipv6_addr_t *src,
+                       ipv6_addr_t *dst, uint16_t len)
 {
     /* TODO handle Solicited Information Option */
     (void) dis;
     (void) len;
+    (void) iface;
```

Comment:
ok, I see we have no convention on this (I mixed it up with stuff from work)
Just forget my whitespace terrror